articles/azure-functions/functions-create-first-java-maven.md

```diff
@@ -24,9 +24,8 @@ To develop functions using Java, you must have the following installed:
 - [Apache Maven](https://maven.apache.org), version 3.0 or above
 - [Azure CLI]
 - [Azure Functions Core Tools](./functions-run-local.md#v2) version 2.6.666 or above
-- An Azure subscription.
```

Comment:
Because the header describing the bullets indicates that you "must have the following installed", I moved out "An Azure subscription" out of the bullets, since you don't install an Azure subscription.